test/intl402/DurationFormat/prototype/format/branding.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype.format
+description: Verifies the branding check for the "format" function of the DurationFormat prototype object.
+features: [Intl.DurationFormat]
+---*/
+
+const format = Intl.DurationFormat.prototype.format;
+
+assert.sameValue(typeof format, "function");
+
+assert.throws(TypeError, () => format.call(undefined), "undefined");
```

Comment:
These are gonna throw anyway even if there's no brand check, because you're not passing a valid object as the duration argument. Compare 9762bc991f9f9492ccd1b28280bceb41c4665f56.